tests/high_performance_vm_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	ktests "kubevirt.io/kubevirt/tests"
 )
 
-var _ = Describe("High performance vm test", func() {
+var _ = Describe("[rfe_id:609, crit:medium, vendor:cnv-qe@redhat.com, level:component]High performance vm test", func() {
```

Comment:
The component level gives us a hint that we should probably move these tests to kubevirt repo
(But doesn't have to be immediately if course, just long term)